.github/workflows/auto_bump_test_package_versions.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           base: master
           title: "[Test Package Versions Bump] Updating package versions "
           milestone: "${{steps.rename.outputs.milestone}}"
-          reviewers: "DataDog/apm-dotnet"
+          team-reviewers: "DataDog/apm-dotnet,DataDog/apm-idm-dotnet"
```

Comment:
Documentation for this is here: https://github.com/peter-evans/create-pull-request?tab=readme-ov-file#action-inputs

This isn't super important as CODEOWNERS kicks in, but it'd be nice to get IDM on there.